src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsoneplanapi/plan/PlanController.kt

```diff
@@ -0,0 +1,158 @@
+package uk.gov.justice.digital.hmpps.hmppsoneplanapi.plan
+
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.media.Content
+import io.swagger.v3.oas.annotations.media.Schema
+import io.swagger.v3.oas.annotations.responses.ApiResponse
+import io.swagger.v3.oas.annotations.tags.Tag
+import kotlinx.coroutines.flow.Flow
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.DeleteMapping
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.justice.digital.hmpps.hmppsoneplanapi.config.ErrorResponse
+import uk.gov.justice.digital.hmpps.hmppsoneplanapi.exceptions.NotFoundException
+import java.util.UUID
+
+@RestController
+@RequestMapping
+@Tag(name = "Plan", description = "Manage plans")
+class PlanController(val planRepository: PlanRepository) {
```

Comment:
Do we have a separate ticket for auth?

Reply:
No, but I was trying to keep the PR non-huge. I can add that to this one if you prefer